clang/docs/ReleaseNotes.rst

```diff
@@ -1076,6 +1076,8 @@ Arm and AArch64 Support
   in leaf functions after enabling ``-fno-omit-frame-pointer``, you can do so by adding
   the ``-momit-leaf-frame-pointer`` option.
 
+- For ARM baremetal targets, the frame pointer (FP) is now turned off by default.
```

Comment:
I would move this before the one above and add to this "To enable the frame pointer use the `-fno-omit-frame-pointer` option. I know it's stated again in the second one, but it answers the "how do I get the old way" question more obviously.

If I am not someone who ever cared to disable FP, I won't understand / care to understand what the leaf function stuff means.

Reply:
done